src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -131,4 +160,22 @@ class EndpointConnectionStats(
                 "timestamp ${reportBlock.lastSrTimestamp}" }
         }
     }
+
+    private fun processTcc(tccPacket: RtcpFbTccPacket, lossTracker: LossTracker) = synchronized(lock) {
```

Comment:
Assuming this is the code you were referring to, I don't think this is bad as-is.